configuration/go/sdk/order-processor/app.go

```diff
@@ -46,13 +46,7 @@ func main() {
 	}
 	fmt.Println("App subscribed to config changes with subscription id: " + subscriptionID)
 
-	time.Sleep(10 * time.Second)
+	time.Sleep(5 * time.Second)
 
-	err = client.UnsubscribeConfigurationItems(context.Background(), DAPR_CONFIGURATION_STORE, subscriptionID)
-	if err != nil {
-		fmt.Println("Error unsubscribing to config updates, err:" + err.Error())
-	} else {
-		fmt.Println("App unsubscribed to config changes")
-	}
```

Comment:
why is this being removed?
Is this removed from SDK or not a valid operation anymore?

Reply:
It has been deprecated in runtime https://github.com/dapr/dapr/blob/9105124636459e751964f3a68845fc63ccbb45e9/pkg/grpc/api.go#L1459

It should be marked as such in SDKs (I don't think it is atm anywhere).

Reply:
I agree it should be removed ... but is it causing an issue ... 

On a side note, @dapr/maintainers-quickstarts When will the latest SDK RCs will be tested for quoickstarts.

Reply:
> I agree it should be removed ... but is it causing an issue ...

Yes, with it included the test runner is printing out an error saying that the subscription cannot be unsubscribed because it doesn't exist. That is a true error- the subscription at the top of the file is given a context with a timeout of 10 seconds, we then do some work, then sleep 10 seconds. This means that when we go to unsubscribe here, the subscription context has been cancelled and closed, and therefore unsubscribed. The unsubscribe here now makes now sense.

The relevant runtime PR: https://github.com/dapr/dapr/pull/6769

Reply:
ok that is marked as a breaking change ....

Reply:
Also the other question, is the unsubscribe method only failing in GO SDK? 
Does it fail in any other SDK?

Reply:
@mukundansundar I assume it is not failing in other quickstart tests because the logic is different. For example, JS is doing the right thing here by just closing the stream https://github.com/dapr/quickstarts/blob/e278ba420f811eb87c2e41f911c7c7b04afa2792/configuration/javascript/sdk/order-processor/index.js#L24

Reply:
[I opened a PR in the go-sdk repo](https://github.com/dapr/go-sdk/pull/457)

Reply:
I believe the context timeout was also causing an issue due to us having a context with a timeout, then sleeping for that duration.

Reply:
good catch